### PR TITLE
Add jms 2.0 dependency.

### PR DESCRIPTION
**Pull request by inthirakumaaran**

### CODE REVIEW

File: components/jms-mq-support/jms-api-orbit/pom.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  ~ under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.wso2.carbon.cache.sync.manager</groupId>
+    <artifactId>jms-api-orbit</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
```
Comment (pamodaaw) [inline components/jms-mq-support/jms-api-orbit/pom.xml]: Is this version correct

File: components/jms-mq-support/jms-api-orbit/2.0.1.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.orbit.org.apache.jm.api</groupId>
```
Comment (pamodaaw) [inline components/jms-mq-support/jms-api-orbit/2.0.1.wso2v1/pom.xml]:
```suggestion
    <groupId>org.wso2.orbit.org.apache.jms.api</groupId>
```

File: components/jms-mq-support/jms-api-orbit/2.0.1.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    <artifactId>jms-api_2</artifactId>
+    <packaging>bundle</packaging>
+    <name>javax.jms.api.2.0.1 orbit bundle</name>
+    <version>2.0.1.wso2v1</version>
+    <description>
+        org.wso2.carbon.ant. This bundle will represent Ant
```
Comment (pamodaaw) [inline components/jms-mq-support/jms-api-orbit/2.0.1.wso2v1/pom.xml]: Is this correct?

File: components/jms-mq-support/jms-api-orbit/pom.xml (hunk 1 above)
Comment (pamodaaw) [inline components/jms-mq-support/jms-api-orbit/pom.xml]: check the version

File: components/jms-mq-support/jms-api-orbit/2.0.1.wso2v1/pom.xml (hunk 3 above)
Comment (inthirakumaaran) [reply components/jms-mq-support/jms-api-orbit/2.0.1.wso2v1/pom.xml]: Fixed

File: components/jms-mq-support/jms-api-orbit/2.0.1.wso2v1/pom.xml (hunk 2 above)
Comment (inthirakumaaran) [reply components/jms-mq-support/jms-api-orbit/2.0.1.wso2v1/pom.xml]: Need to change the whole Id

**Review by pamodaaw — APPROVED** [review]

**Pull request merged**